pants

```diff
@@ -45,12 +43,30 @@ function tempdir {
   mktemp -d "$1"/pants.XXXXXX
 }
 
+function get_pants_ini_config_value {
+  config_key="$1"
+  valid_delimiters="[:=]"
+  optional_space="[[:space:]]*"
+  prefix="^${config_key}${optional_space}${valid_delimiters}${optional_space}"
+  remove_prefix=" s#${prefix}##p"
+  sed -ne "/${prefix}/${remove_prefix}" pants.ini
+}
+
+function determine_pants_runtime_python_version {
+  python_bin_name="${PYTHON:-python2.7}"
+  if which "${python_bin_name}" >/dev/null; then
+    echo "$(which ${python_bin_name})"
+  else
+    echo "${python_bin_name} could not be found on your PATH! Pants will not work." 1>&2
+    exit 1
+  fi
```

Comment:
This is readable enough, but could also be:
```bash
  if ! which "${python_bin_name}"; then
    echo "${python_bin_name} could not be found on your PATH! Pants will not work." 1>&2
    exit 1
  fi
```
It will produce an extra `<x> not found` line in the error case, but I think that's ok / what we want here.

Reply:
Yes I like this more. Thanks.

Reply:
With the above, I was actually thinking you could remove the `>/dev/null` and also remove the `echo` line below the `if`.